examples/python/quickstart_sql.py

```diff
@@ -0,0 +1,74 @@
+
+from pyspark.sql import Column, DataFrame, SparkSession, functions
+from pyspark.sql.functions import *
+from py4j.java_collections import MapConverter
+from delta.tables import *
+import shutil
+import threading
+
+tableName = "tbltestpython"
+
+# Enable SQL/DML commands and Metastore tables for the current spark session.
+# We need to set the following configs 
+
+spark = SparkSession.builder \
+    .appName("quickstart") \
```

Comment:
Maybe we can rename the appName to be `quickstart_sql` here just to be consistent.